### PR TITLE
Clever Cloud documentation

### PR DESCRIPTION
**Pull request by RudyBaer**
Give information for using Clever Cloud with jhipster

- [x] information about Clever Cloud
- [x] Clever Cloud logo
- [x] Before you start
- [x] How to create Clever Cloud app
- [x] How to configure jhipster application
  - [x] with maven
  - [x] with gradle
  - [x] for mysql
  - [x] for postgresql
  - [x] for mongo
- [x] How deploy
  - [x] using CLI
  - [x] using CI
- [x] How to change java version
- [x] More information

old resource :
- [x] https://github.com/CleverCloud/generator-jhipster-clevercloud

### CODE REVIEW

**Comment by cbornet** [comment]
Could this be done as a generator like we have for the other clouds ?

**Comment by RudyBaer** [comment]
> Could this be done as a generator like we have for the other clouds ?

yes @cbornet  
there is an [old generator](https://github.com/CleverCloud/generator-jhipster-clevercloud) made by clever cloud.
Last time I used it, it had some trouble. I need to check again

**Comment by atomfrede** [comment]
It's based on jhipster 4 so it won't work. From the documentation it seems similar to what we have for the core heroku subgen. So integration into main generator should be possible.

File: pages/clever-cloud.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    spring:
+      data:
+        mongodb:
+          uri: ${MONGODB_ADDON_URI}
+          database: ${MONGODB_ADDON_DB}
+    </pre>
+
+
+
+3. add json file
```
Comment (atomfrede) [inline pages/clever-cloud.md:97]: maybe add one sentence whats the purpose of this json file (like for the .yml file)

Comment (RudyBaer) [reply pages/clever-cloud.md]: > add a json file that contain the goal field to indicate how to start your application
is it better ?

**Pull request merged**